### PR TITLE
End Of Year: Implement Design Notes Feedback for the Intro Story

### PR DESCRIPTION
**Pull request by emilylaguna**
| 📘 Project: #376 |
|:---:|

#### Text Changes:
- Increases line height to 2.5 to better match the design
- Replaces typewriter apostrophes `'` with typographic ones `’`
- Replaces `...` with `!`
- Inserts a newline between `...your year` and `of ...` to prevent widows

#### UI Changes
- Moves the 2022 logo down, it is now 10% from the top of the view
- Reduces the logo bottom padding from 40 to 24
   - This applies to all stories
- Increased share button leading/trailing padding to 20 from 5
    - This applies to all stories, not seen in below screenshots

#### Code changes
- Created a new common view for the logo and applied it to all stories

### Design vs Implementation Comparison
<p align="center"></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
||||

## To test

1. Launch the app
2. Go to the Profile Tab
3. Tap on the End of Year Card
4. ✅ Verify the Intro Story matches the designs (See comparison above)
7. Tap through to the other stories
8. ✅ Verify the Logo appears in a consistent location on each of them (except for the final view, will be updated later)
5. Launch the app on various device sizes
6. ✅ Verify the view still looks good on all of them

## Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: podcasts/End of Year/Stories/Views/PodcastCover.swift [hunk 1]
```diff
@@ -95,10 +95,10 @@ struct NormalCoverShadow: ViewModifier {
...
         content
             .cornerRadius(4)
-            .shadow(color: .black.opacity(0.1), radius: 1, x: 0, y: 1)
-            .shadow(color: .black.opacity(0.09), radius: 3, x: 0, y: 3)
-            .shadow(color: .black.opacity(0.05), radius: 4, x: 0, y: 6)
-            .shadow(color: .black.opacity(0.01), radius: 4, x: 0, y: 11)
+            .shadow(color: .black.opacity(0.1 + 0.2), radius: 1, x: 0, y: 1)
+            .shadow(color: .black.opacity(0.09 + 0.2), radius: 3, x: 0, y: 3)
+            .shadow(color: .black.opacity(0.05 + 0.2), radius: 4, x: 0, y: 6)
+            .shadow(color: .black.opacity(0.01 + 0.2), radius: 4, x: 0, y: 11)
```
Comment (jgcaruso) [inline podcasts/End of Year/Stories/Views/PodcastCover.swift]: can `0.2` be a constant somewhere?

File: podcasts/End of Year/Stories/Views/PodcastCover.swift [hunk 2]
```diff
@@ -108,10 +108,10 @@ struct BigCoverShadow: ViewModifier {
...
         content
             .cornerRadius(8)
-            .shadow(color: .black.opacity(0.1), radius: 9, x: 0, y: 4)
-            .shadow(color: .black.opacity(0.09), radius: 17, x: 0, y: 17)
-            .shadow(color: .black.opacity(0.05), radius: 23, x: 0, y: 38)
-            .shadow(color: .black.opacity(0.01), radius: 27, x: 0, y: 67)
+            .shadow(color: .black.opacity(0.1 + 0.2), radius: 9, x: 0, y: 4)
+            .shadow(color: .black.opacity(0.09 + 0.2), radius: 17, x: 0, y: 17)
+            .shadow(color: .black.opacity(0.05 + 0.2), radius: 23, x: 0, y: 38)
+            .shadow(color: .black.opacity(0.01 + 0.2), radius: 27, x: 0, y: 67)
```
Comment (jgcaruso) [inline podcasts/End of Year/Stories/Views/PodcastCover.swift]: can `0.2` be a constant somewhere?

**Review by jgcaruso — CHANGES_REQUESTED** [review]
Code changes look good (I'm assuming the ShareableStore code matches https://github.com/Automattic/pocket-casts-ios/pull/555 so I didn't scrutinize it too much this time) and the Intro screen on device matches the design.

Something is up with smaller screens though (iPhone 6S and SE but not on 13 mini ). This PR vs #555 causes an overlap on the `TopFivePodcastsStory` screen

| before (#555) | after (this pr) |
|-----|-----|
| IMG_0008 |  IMG_0007 |

File: podcasts/End of Year/Stories/Views/PodcastCover.swift (hunk 2 above)
Comment (emilylaguna) [reply podcasts/End of Year/Stories/Views/PodcastCover.swift]: Oops. That wasn't meant to be committed. Reverted it here: ba7b640a5898c13a9a5a6b6013c5429308bdb034

File: podcasts/End of Year/Stories/Views/PodcastCover.swift (hunk 1 above)
Comment (emilylaguna) [reply podcasts/End of Year/Stories/Views/PodcastCover.swift]: Oops. That wasn't meant to be committed. Reverted it here: ba7b640a5898c13a9a5a6b6013c5429308bdb034

**Comment by emilylaguna** [comment]
> Something is up with smaller screens though (iPhone 6S and SE but not on 13 mini ). This PR vs #555 causes an overlap on the `TopFivePodcastsStory` screen

Yeah, this is because the share logo is absolutely positioned over the entire view. I'm planning on fixing that screen in a future pr.

**Review by jgcaruso — APPROVED** [review]
Latest change and comments re: logo LGTM!

**Pull request merged**